swagger-express-mw/index.d.ts

```diff
@@ -0,0 +1,44 @@
+// Type definitions for swagger-express-mw 0.7
+// Project: https://github.com/apigee-127/swagger-express#readme
+// Definitions by: Michael Mrowetz <https://github.com/micmro/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/* =================== USAGE ===================
+
+import * as SwaggerExpress from "swagger-express-mw";
+import * as express from "express";
+
+let app = express();
+let config: SwaggerExpress.Config = {
+    appRoot: __dirname
+};
+
+SwaggerExpress.create(config, (err, middleware) => {
+    if (err) {
+        throw err; // or handle error
+    }
+    middleware.register(app);
+
+    const port = process.env.PORT || 10010;
+    app.listen(port);
+});
+
+ =============================================== */
+
+import { Config, ConnectMiddleware } from "../swagger-node-runner";
```

Comment:
Test failure refers to this line. Use a global import from `"swagger-node-runner"`.